include/mscclpp/concurrency_device.hpp

```diff
@@ -51,11 +46,11 @@ struct DeviceSyncer {
 
  private:
   /// The flag to indicate whether the barrier is reached by the latest thread.
-  volatile int flag_;
+  unsigned int flag_;
   /// The counter of synchronized blocks.
   unsigned int count_;
   /// The flag to indicate whether to increase or decrease @ref flag_.
-  int isIncFlag_;
+  unsigned int isIncFlag_;
```

Comment:
Change to `preFlag_`? Seems this flag is used to indicate the flag value in last round.

Reply:
Fixed by [c1f3f2b](https://github.com/microsoft/mscclpp/pull/231/commits/c1f3f2b77e0a1b8d4b4cf095f9333265e49b6872)